test/uk/gov/hmrc/nisp/views/ExclusionViewSpec.scala

```diff
@@ -84,8 +85,8 @@ class ExclusionViewSpec extends HtmlSpec with MockitoSugar {
 
   "Exclusion Isle of Man : Can't see NI Record: State Pension Age under consideration - no flag" should {
 
-    lazy val sResult = html.excluded_sp(Exclusion.IsleOfMan, Some(40), Some(new LocalDate(2019, 9, 6)), false, None)
-    lazy val htmlAccountDoc = asDocument(contentAsString(sResult))
+    lazy val sResult = inject[excluded_sp]
+    lazy val htmlAccountDoc = asDocument(sResult.apply(Exclusion.IsleOfMan, Some(40), Some(LocalDate.of(2019, 9, 6)), false, None).toString)
```

Comment:
Don't need `.apply` here. calling `someObject.apply()` is equivalent  to `someObject()`
Same in the rest of the file